File: SofiForce.Models/Models/DtoModels/TrakingRepresentativeDetailModel.cs
using System;

namespace Models
{
    public class TrakingRepresentativeDetailModel
    {
        public long TrackingId { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public bool? IsPositive { get; set; }
        
        public string ClientCode { get; set; }
        public string ClientName { get; set; }
        public string TrackTypeName { get; set; }
        public int TrackTypeId { get; set; }

        public double? SalesId { get; set; }
        public int? ClientId { get; set; }
        public DateTime? TrackTime { get; set; }

        public bool? InZone { get; set; }
        public int Distance { get; set; }


    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/TrakingRepresentativeDetailModel.ts
export class TrakingRepresentativeDetailModel {
    trackingId: number;
    latitude: number | null;
    longitude: number | null;
    isPositive: boolean | null;
    inZone: boolean | null;
    distance: number | null;

    clientCode: string;
    clientName: string;
    trackTypeName: string;
    trackTypeId: number;
    salesId: number | null;
    clientId: number | null;
    trackTime:Date;
}